packages/calcite-components/src/components/combobox/combobox.stories.ts

```diff
@@ -449,17 +449,31 @@ export const withSelectorIndicatorAndIcons_TestOnly = (): string => html`
   </calcite-combobox>
 `;
 
-export const nestedGroups_TestOnly =
-  (): string => html`<calcite-combobox open selection-mode="single" style="width:400px" placeholder="Type to filter">
-<calcite-combobox-item-group label="Level 1">
-  <calcite-combobox-item-group label="Level 2">
-    <calcite-combobox-item-group label="Level 3">
-      <calcite-combobox-item value="Item 1" text-label="Item 1">
-      </calcite-combobox-item>
-      <calcite-combobox-item value="Item 2" text-label="Item 2">
+export const nestedGroups = (): string => html`
```

Comment:
Should this still use the `_TestOnly` suffix? This is not critical for this PR and can be tackled separately.

Reply:
yep, let me fix that before this merge